<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="myfile" name="myfile_page"/>
      <widget class="ShareFileWidget" name="share_page"/>
      <widget class="QWidget" name="load_page"/>
      <widget class="transformwidget" name="tran_page"/>
      <widget class="QWidget" name="switch_page"/>
      <widget class="QWidget" name="user_page"/>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="buttongroup" name="butgroup" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>buttongroup</class>
   <extends>QWidget</extends>
   <header>buttongroup.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>myfile</class>
   <extends>QWidget</extends>
   <header>myfile.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>transformwidget</class>
   <extends>QWidget</extends>
   <header location="global">transformwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ShareFileWidget</class>
   <extends>QWidget</extends>
   <header location="global">sharefilewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
